src/views/v2/month/event-multiday.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
It'd be good to go ahead and keep a `<time datetime="...">...</time>` element here and a11y visually hide it. Just be consistent in your html implementation of it between this and the events for a given day.